reference/events.rst

```diff
@@ -77,7 +77,7 @@ This is the built-in Symfony listener related to this event:
 ==============================================================================  ========
 Listener Class Name                                                             Priority
 ==============================================================================  ========
-:class:`Symfony\\Bundle\\FrameworkBundle\\DataCollector\\RequestDataCollector`  0
```

Comment:
I don't know why this class is here, but it was added in 3.1 with https://github.com/symfony/symfony/pull/17589. Does it worth reverting it in master?